src/components/AvatarWithImagePicker.js

```diff
@@ -293,95 +317,94 @@ class AvatarWithImagePicker extends React.Component {
                                                 if (index === 0 && Browser.isSafari()) {
                                                     openPicker({
-                                                        onPicked: this.showAvatarCropModal,
+                                                        onPicked: showAvatarCropModal,
                                                     });
                                                 }
                                             }}
                                             menuItems={menuItems}
-                                            anchorPosition={this.props.anchorPosition}
+                                            anchorPosition={anchorPosition}
                                             withoutOverlay
-                                            anchorRef={this.anchorRef}
-                                            anchorAlignment={this.props.anchorAlignment}
+                                            anchorRef={anchorRef}
+                                            anchorAlignment={anchorAlignment}
                                         />
                                     );
                                 }}
                             </AttachmentPicker>
                         )}
                     </AttachmentModal>
                 </View>
-                {this.state.validationError && (
+                {errorData.validationError && (
                     <DotIndicatorMessage
                         style={[styles.mt6]}
-                        messages={{0: this.props.translate(this.state.validationError, this.state.phraseParam)}}
+                        messages={{0: translate(errorData.validationError, errorData.phraseParam)}}
                         type="error"
                     />
                 )}
                 <AvatarCropModal
-                    onClose={this.hideAvatarCropModal}
-                    isVisible={this.state.isAvatarCropModalOpen}
-                    onSave={this.props.onImageSelected}
-                    imageUri={this.state.imageUri}
-                    imageName={this.state.imageName}
-                    imageType={this.state.imageType}
-                    maskImage={this.props.editorMaskImage}
+                    onClose={hideAvatarCropModal}
+                    isVisible={isAvatarCropModalOpen}
+                    onSave={onImageSelected}
+                    imageUri={imageData.uri}
+                    imageName={imageData.name}
+                    imageType={imageData.type}
+                    maskImage={editorMaskImage}
                 />
-            </View>
-        );
-    }
+        </View>
+    );
 }
 
 AvatarWithImagePicker.propTypes = propTypes;
 AvatarWithImagePicker.defaultProps = defaultProps;
 
-export default compose(withLocalize, withNavigationFocus)(AvatarWithImagePicker);
+export default withNavigationFocus(AvatarWithImagePicker);
```

Comment:
Use `const isFocused = useIsFocused();` hook instead of `withNavigationFocus` HOC.

Most likely it will cause app to crash due to this [bug](https://github.com/Expensify/App/pull/23160#issuecomment-1673302723).

Reply:
yes, will keep it